ui/summary_test.go

```diff
@@ -164,3 +166,125 @@ func createTestMetrics() map[string]*stats.Metric {
 
 	return metrics
 }
+
+func TestSummarizeMetricsJSON(t *testing.T) {
+	metrics := createTestMetrics()
+	expected := `[{
+    "name": "",
+    "path": "",
+    "id": "d41d8cd98f00b204e9800998ecf8427e",
+    "groups": {
+        "child": {
+        "name": "child",
+        "path": "::child",
+        "id": "f41cbb53a398ec1c9fb3d33e20c9b040",
+        "groups": {},
+        "checks": {
+            "check1": {
+                "name": "check1",
+                "path": "::child::check1",
+                "id": "6289a7a06253a1c3f6137dfb25695563",
+                "passes": 5,
+                "fails": 10
+                }
+            }
+        }
+    },
+    "checks": {}
+},
+{
+    "checks": {
+        "extra": [
+            "✓ 3",
+            "✗ 0"
```

Comment:
I don't think this should look like this .. it should probably be:
```
{"passed": 3, "failed": 0}
```

Reply:
Same as https://github.com/loadimpact/k6/pull/1168/files#r343545942